### ISSUE TITLE
python2-{babel,sphinx} postinstall broken

### ISSUE THREAD

**Issue by Optiligence** [Alexpux/MINGW-packages#3412]
```
( 3/12) upgrading mingw-w64-x86_64-python2-babel   [#####################] 100%
/
sed: can't read mingw64/bin/pybabel-script.py: No such file or directory
error: command (/usr/bin/bash /usr/bin/bash -c . /tmp/alpm_OAvPit/.INSTALL; post_upgrade 2.5.3-1 2.5.1-1 ) failed to execute correctly
```
[…]
```
( 9/12) upgrading mingw-w64-x86_64-python2-sphinx  [#####################] 100%
/
sed: can't read mingw64/bin/sphinx-apidoc-script.py: No such file or directory
sed: can't read mingw64/bin/sphinx-autogen-script.py: No such file or directory
sed: can't read mingw64/bin/sphinx-build-script.py: No such file or directory
sed: can't read mingw64/bin/sphinx-quickstart-script.py: No such file or directory
error: command (/usr/bin/bash /usr/bin/bash -c . /tmp/alpm_KhWy6S/.INSTALL; post_upgrade 1.7.0-1 1.6.6-1 ) failed to execute correctly
```

**Comment by lazka** [comment]
@JPeterMugaas

**Comment by JPeterMugaas** [comment]
I am aware of the issue and believe someone else had fixed it.  It was probably an error I made when copying text from another PKGBUILD file.  

The way I see the Python packages is as a pattern where certain steps are done and I try to repeat those in a standard form.  That's why the packages have the pver loops as well as fixup install scripts.  If anything, I would like to see every one of the python packages conform the pattern.

**Comment by lazka** [comment]
If you would change the shebang to "#!python3.exe" you wouldn't need any of this as it would use the script directory + executable name: https://github.com/pypa/setuptools/blob/9d12348e26d8c143d0b675ed147094a9e54e1c97/launcher.c#L119

**Comment by lazka** [comment]
I guess this was fixed in dfc398f83f243083e3b39ab

Thanks @Optiligence
